content/documentation/glossary/networking/index.adoc

```diff
@@ -0,0 +1,67 @@
+---
+title: "Networking"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation/glossary"
+---
+
+= Kiali Networking Glossary
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Networking Glossary Content
+:keywords: Kiali Documentation Glossary
+:icons: font
+
+== Destination
+
+Read the <<Destination>> definition in link:https://istio.io/help/glossary/#destination[Istio Glossary]
+
+=== Destination Rule
+
+Read the <<Destination Rule>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#DestinationRule[Istio Destination Rule]
```

Comment:
`Read the` could be replaced with `For more information see the` as it just sounds more polite.